### PR TITLE
Updated benchmarks to work on latest Django main

### PR DESCRIPTION
**Pull request by collinanderson**
Updated benchmarks to work on latest Django main

### CODE REVIEW

File: djangobench/benchmarks/default_middleware/urls.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 try:
-    from django.conf.urls import url
+    from django.urls import re_path as url
```
Comment (adamchainz) [inline djangobench/benchmarks/default_middleware/urls.py:2]: can we go the other way and import `url as re_path` for older versions?

Also maybe better to use version checking - `if django.VERSION >= (2, 0):` then import `re_path`

https://adamj.eu/tech/2020/01/07/feature-checking-versus-version-checking/

**Review by adamchainz — COMMENTED** [review]
Thanks for reviving the benchmarks :)

File: djangobench/benchmarks/default_middleware/urls.py (hunk 1 above)
Comment (collinanderson) [reply djangobench/benchmarks/default_middleware/urls.py]: Ok I switched everything from `url()` to `re_path()` and used version checking.

**Comment by adamchainz** [comment]
Thanks!

**Pull request merged**